conf/messages

```diff
@@ -4518,6 +4518,14 @@ multipleDisposalsDisposalPrice.personalRepInPeriodOfAdmin.agent.helpText=This is
 multipleDisposalsDisposalPrice.link=What are disposal costs?
 multipleDisposalsDisposalPrice.details.p1=Disposal or selling costs include fees for estate agents or solicitors.
 
+multipleDisposalsDisposalPrice.input=What was the disposal value of the property?
```

Comment:
could probably collapse this into a single message entry

Reply:
I thought so too but that's the way its done in the message file so I adjusted to that to maintain consistency